test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetInstallCommandTest.cs

```diff
@@ -2239,6 +2240,61 @@ public async Task InstallCommand_WithASourceThatReportsVulnerabilities_RaisesVul
             File.Exists(a1Nupkg).Should().BeFalse();
         }
 
+        [SkipMono()]
+        public void InstallCommand_WithPackageWithVulnerabilities_RaisesWarnings()
+        {
+            // Arrange
+            var nugetexe = Util.GetNuGetExePath();
+            using var pathContext = new SimpleTestPathContext();
+            using var mockServer = new FileSystemBackedV3MockServer(pathContext.PackageSource, sourceReportsVulnerabilities: true);
+
+            mockServer.Vulnerabilities.Add(
+                "packageA",
+                new List<(Uri, PackageVulnerabilitySeverity, VersionRange)> {
+            (new Uri("https://contoso.com/advisories/12345"), PackageVulnerabilitySeverity.High, VersionRange.Parse("[1.0.0, 2.0.0)"))
+                });
+
+            pathContext.Settings.RemoveSource("source");
```

Comment:
I think we need to specify an audit source not a regular source. 

If you delete the changes in INstallCommand & RestoreCOmmand, I think this test still passes.

Another thing, both restore & install are affected, so we should have both an install and a restore test.